hbw/app/javascript/packs/hbw/components/shared/hoc/select.js

```diff
@@ -50,6 +51,12 @@ export default WrappedComponent => class WithSelect extends Component {
     if (((a === '') && (b === null)) || ((a === null) && (b === ''))) {
       return true;
     }
+    if (((a === 'null') && (b === null)) || ((a === null) && (b === 'null'))) {
```

Comment:
:(

Reply:
wtf???

Reply:
There is 'null' value in nullable select tables with multi = false.
    We should parse it like a null
    It will be used in hasValueInChoices function